pkgs/applications/graphics/imv/default.nix

```diff
@@ -17,10 +17,10 @@
 , libGLU
 , wayland
 # "libnsgif" is disabled until https://todo.sr.ht/~exec64/imv/55 is solved
-, withBackends ? [ "freeimage" "libtiff" "libjpeg" "libpng" "librsvg" "libheif" ]
-, freeimage
+, withBackends ? [ "libtiff" "libjpeg" "libpng" "librsvg" "libheif" ]
 , libtiff
 , libjpeg_turbo
+, libjxl
```

Comment:
Should we enable this backend by default in `withBackends` ? :thinking:

Reply:
yes, thanks, done